CONTRIBUTING.md

```diff
@@ -3,24 +3,24 @@
 Interested in contributing to RiveScript? Great!
 
 First, check the general contributing guidelines for RiveScript and its primary
-implementations found at <http://www.rivescript.com/contributing> - in
-particular, understand the goals and scope of the RiveScript language and the
-style guide for the Java implementation.
+implementations found at <http://www.rivescript.com/contributing>*[]: 
```

Comment:
What's this `*[]` part? It's making the code formatting look weird in this PR and doesn't seem to do anything in the rendered Markdown.